fulp_modules/features/clothing/body_cameras/body_camera.dm

```diff
@@ -73,11 +80,13 @@
 	RegisterSignal(parent, COMSIG_ATOM_ATTACKBY, PROC_REF(on_attackby))
 	RegisterSignal(parent, COMSIG_ATOM_EXAMINE_MORE, PROC_REF(on_examine_more))
 	RegisterSignal(parent, COMSIG_ATOM_TOOL_ACT(TOOL_SCREWDRIVER), PROC_REF(on_screwdriver_act))
+	RegisterSignal(parent, COMSIG_ITEM_POST_EQUIPPED, PROC_REF(on_parent_equipped))
```

Comment:
You can probably register this when the camera is being installed instead

Reply:
Done and (slightly) tested— it seems to work still, I think...